aptos-move/framework/aptos-framework/sources/configs/consensus_config.move

```diff
@@ -24,14 +24,15 @@ module aptos_framework::consensus_config {
     }
 
     /// This can be called by on-chain governance to update on-chain consensus configs.
-    public fun set(account: &signer, config: vector<u8>) acquires ConsensusConfig {
+    public fun set(account: &signer, config: vector<u8>) {
         system_addresses::assert_aptos_framework(account);
         assert!(vector::length(&config) > 0, error::invalid_argument(EINVALID_CONFIG));
+        std::config_for_next_epoch::upsert<ConsensusConfig>(account, ConsensusConfig {config});
+    }
```

Comment:
Today, `consensus_config::set()` calls `reconfiguration::reconfigure()`.
But in our new design, `consensus_config::set(X)` writes `X` to a buffer; then at the end of DKG, reconfiguration calls a new func `consensus_config::on_new_epoch()` to apply `X`.
This means that we can't control `consensus_config::set()` behavior with a feature flag, otherwise compile fails with a circular dependency.

But I think if we also update `aptos-release-builder` accordingly to append a `aptos_governance::reconfigure()` in the proposal script, things sound be fine?